.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: Bug
+label: bug
+---
+
+<!--
+Please fill in the *entire* template below. Screenshots and logs will be very helpful.
+-->
+
```

Comment:
Could we categorize the environment info as following?

**Environment**:
- VS Code Version:
- Extension Version:
- JDK Type & Version:
- OS Type & Version: <!-- Windows, Mac, Linux-->

**Step to Reproduce:**
...